File: androidchallenge1/src/main/java/class28/androidchallenge01/ui/restaurants/RestaurantsListViewHolder.java
package class28.androidchallenge01.ui.restaurants;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import class28.androidchallenge01.R;

class RestaurantsListViewHolder extends RecyclerView.ViewHolder {

    ImageView restaurantImage;
    TextView restaurantName, restaurantAddress, restaurantClosesAt;

    RestaurantsListViewHolder(@NonNull View itemView) {
        super(itemView);

        restaurantImage = itemView.findViewById(R.id.restaurant_image);
        restaurantName = itemView.findViewById(R.id.restaurant_name);
        restaurantAddress = itemView.findViewById(R.id.tvRestaurantAddress);
        restaurantClosesAt = itemView.findViewById(R.id.tvRestaurantClosesAt);
    }
}

File: androidchallenge1kotlin/src/main/java/class28/androidchallenge01kotlin/ui/restaurants/RestaurantsListViewHolder.kt
package class28.androidchallenge01kotlin.ui.restaurants

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import class28.androidchallenge01kotlin.R

class RestaurantsListViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var restaurantImage: ImageView = itemView.findViewById(R.id.restaurant_image)
    var restaurantName: TextView = itemView.findViewById(R.id.restaurant_name)
    var restaurantAddress: TextView = itemView.findViewById(R.id.tvRestaurantAddress)
    var restaurantClosesAt: TextView = itemView.findViewById(R.id.tvRestaurantClosesAt)
}